.travis.yml

```diff
@@ -1,12 +1,26 @@
 language: python
 python:
-  - "3.4"
+  - "3.6"
 cache: pip
 install:
   - pip install -r requirements.txt
   - pip install pytest-cov
   - pip install coveralls
+before_script:
+  - export FLASK_APP="run.py"
+  - export SECRET_KEY="!@#(#_jjdril^&@efr8599"
```

Comment:
You should set your environment variables in travis. Pushing this file with the values of your environment variables to a public repo is a security hazard.